app/scripts/views/mixins/last-checked-time-mixin.js

```diff
@@ -21,12 +29,12 @@ const Mixin = {
     });
   },
 
-  _getLastCheckedTimeString() {
-    let time = 'none';
+  getLastCheckedTimeString() {
+    let time = t('none');
     if (this.lastCheckedTime) {
       time = this.lastCheckedTime;
     }
-    return `Last checked: ${time}`;
+    return t('Last checked') + `: ${time}`;
```

Comment:
Localizers get a bit twitchy when we do string concatenation to form strings. I'd suggest doing something like:

```js
return this.translate(t('Last checked: %(lastCheckTime)s'), { lastCheckTime: time );
```